test/export/test_hop.py

```diff
@@ -0,0 +1,148 @@
+# Owner(s): ["oncall: export"]
+# flake8: noqa
+import copy
+import dataclasses
+import io
+import unittest
+import warnings
+from contextlib import contextmanager
+from dataclasses import dataclass
+from re import escape
+
+import torch
+import torch._dynamo as torchdynamo
+import torch.nn.functional as F
+import torch.utils._pytree as pytree
+from functorch.experimental.control_flow import cond, map
+from torch import Tensor
+from torch._dynamo.test_case import TestCase
+from torch._export.pass_base import _ExportPassBaseDeprecatedDoNotUse
+from torch._export.utils import (
+    get_buffer,
+    get_param,
+    is_buffer,
+    is_param,
+    register_dataclass_as_pytree_node,
+)
+from torch._subclasses import FakeTensorMode
+from torch.export import Dim, dynamic_dim, export, unflatten
+from torch.export._trace import (
+    _export,
+    _export_to_torch_ir,
+    DEFAULT_EXPORT_DYNAMO_CONFIG,
+)
+from torch.fx.experimental.proxy_tensor import make_fx
+from torch.testing import FileCheck
+from torch.testing._internal.common_cuda import PLATFORM_SUPPORTS_FLASH_ATTENTION
+from torch.testing._internal.common_device_type import (
+    instantiate_device_type_tests,
+    onlyCPU,
+    onlyCUDA,
+    ops,
+)
+from torch.testing._internal.common_utils import (
+    find_library_location,
+    IS_FBCODE,
+    IS_MACOS,
+    IS_SANDCASTLE,
+    IS_WINDOWS,
+    run_tests,
+    TestCase as TorchTestCase,
+)
+from torch.testing._internal.hop_exportability_db import (
+    hop_export_opinfo_db,
+    hop_that_doesnt_need_export_support,
+)
+
+try:
+    from . import testing
```

Comment:
Not related to this PR but curious why we need to try-except but I don't see test for other components doing this?

Reply:
I just copied it from test_export.py. Likely not needed.

Reply:
Dynamo test will also run test internally, right? but i didn't see them doing this or is it hidden somewhere?

Reply:
Oh this was because doing `import testing` doesn't work in fbcode, so we need to do `from . import testing`

Reply:
But it looks like you're not using the `testing` module anywhere here so I don't think you need this import?
The `testing` module has stuff like `testing.expectedFailureRetraceability`, `testing.expectedFailureNonStrict`